### PR TITLE
[prometheus] distroless prometheus

### PR DESCRIPTION
**Pull request by aleksey-su**
## Description

Prometheus are now based on minimal image.

## Why do we need it, and what problem does it solve?

We'd like to base our images on a distroless image.

## What is the expected result?

We use scratch along with a minimal number of files from Alpine packages. We pass files in packages though an rsync whitelist.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: prometheus
type: feature
summary: Images are based on a distroless image.
impact_level: default
```

### CODE REVIEW

**Comment by aleksey-su** [comment]
image
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

File: modules/300-prometheus/images/prometheus/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  stageDependencies:
+    install:
+    - '**/*.patch'
+shell:
+  beforeInstall:
+  - |
+    apt-get update && apt-get install -y make bash git ca-certificates openssl \
+      openssh-client bzip2 curl gnupg apt-utils \
+    && curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/trusted.gpg.d/nodesource.gpg \
+    && NODE_MAJOR=20 \
```
Comment (RomanenkoDenys) [inline modules/300-prometheus/images/prometheus/werf.inc.yaml:44]:
```suggestion
    - curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/trusted.gpg.d/nodesource.gpg
    - export NODE_MAJOR=20
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
Please do not use &&

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Pull request merged**